packages/babel-core/src/config/config-chain.ts

```diff
@@ -582,8 +582,12 @@ function makeChainWalker<
     // that we don't do extra work loading extended configs if a file is
     // ignored.
     if (
-      flattenedConfigs.some(({ config: { options: { ignore, only } } }) =>
-        shouldIgnore(context, ignore, only, dirname),
+      flattenedConfigs.some(
+        ({
+          config: {
+            options: { ignore, only },
+          },
+        }) => shouldIgnore(context, ignore, only, dirname),
```

Comment:
I think the previous one was a bit more readable, since this new version gives way more importance to the destructuring than to the actual arrow body.